### DISCUSSION TITLE
hello please help me

### DISCUSSION THREAD

**Discussion by tushar158gupta** [vercel/next.js#27435] [Help]
<html>
<body>

17:31:17.969 | Cloning github.com/Tushar-gupta-git/Whatsapp (Branch: main, Commit: e0d9258)
-- | --
17:31:18.733 | Cloning completed: 764.735ms
17:31:18.756 | Analyzing source code...
17:31:19.723 | Installing build runtime...
17:31:22.847 | Build runtime installed: 3.123s
17:31:25.211 | No Build Cache available
17:31:26.300 | Installing dependencies...
17:31:26.647 | yarn install v1.22.10
17:31:26.699 | warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
17:31:26.702 | [1/4] Resolving packages...
17:31:30.354 | [2/4] Fetching packages...
17:31:48.695 | info fsevents@2.3.2: The platform "linux" is incompatible with this module.
17:31:48.695 | info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
17:31:48.699 | [3/4] Linking dependencies...
17:31:48.703 | warning " > styled-components@5.3.0" has unmet peer dependency "react-is@>= 16.8.0".
17:31:48.705 | warning "eslint-config-next > @typescript-eslint/parser > @typescript-eslint/typescript-estree > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 \|\| >= 3.2.0-dev \|\| >= 3.3.0-dev \|\| >= 3.4.0-dev \|\| >= 3.5.0-dev \|\| >= 3.6.0-dev \|\| >= 3.6.0-beta \|\| >= 3.7.0-dev \|\| >= 3.7.0-beta".
17:32:06.518 | [4/4] Building fresh packages...
17:32:06.995 | success Saved lockfile.
17:32:06.999 | Done in 40.36s.
17:32:07.043 | Detected Next.js version: 11.0.1
17:32:07.045 | Running "yarn run build"
17:32:07.317 | yarn run v1.22.10
17:32:07.343 | $ next build
17:32:08.073 | info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
17:32:08.322 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
17:32:08.322 | This information is used to shape Next.js' roadmap and prioritize features.
17:32:08.322 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
17:32:08.322 | https://nextjs.org/telemetry
17:32:08.412 | info  - Checking validity of types...
17:32:09.712 | info  - Creating an optimized production build...
17:32:12.494 | Failed to compile.
17:32:12.494 | ModuleNotFoundError: Module not found: Error: Can't resolve '../components/Sidebar' in '/vercel/path0/pages'
17:32:12.494 | > Build error occurred
17:32:12.495 | Error: > Build failed because of webpack errors
17:32:12.495 | at /vercel/path0/node_modules/next/dist/build/index.js:15:924
17:32:12.495 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:6:584)
17:32:12.515 | error Command failed with exit code 1.
17:32:12.515 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
17:32:12.545 | Error: Command "yarn run build" exited with 1

</body>
</html>

**Comment by ijjk** [comment]
Hi, this looks related to https://github.com/vercel/next.js/discussions/27372 and seems to be from the casing being incorrect. See [the comment](https://github.com/vercel/next.js/discussions/27372#discussioncomment-1033766) specifically for tips on fixing the casing in git
